src/DynamoCoreWpf/ViewModels/Search/SelectionNavigator.cs

```diff
@@ -61,8 +65,12 @@ internal void UpdateRootCategories(IEnumerable<SearchCategory> rootTree)
             selectedMemberGroupIndex = 0;
             selectedMemberIndex = 0;
 
-            selection = GetSelectionFromIndices();
-            selection.IsSelected = true;
+            selection = null;
+        }
+
+        internal void UpdateTopResult(NodeSearchElementViewModel topRes)
+        {
+            topResult = topRes;
         }
```

Comment:
We should add `topResult` to `UpdateRootCategories` method, and remove `UpdateTopResult` method (because updating root and setting top result should always be done together):

```csharp
internal void UpdateRootCategories(IEnumerable<SearchCategory> rootTree, NodeSearchElementViewModel topResult)
{
    root = rootTree;

    if (root == null || !root.Any())
    {
        selection = null;
        topResult = null;
        return;
    }

    selectedCategoryIndex = -1;
    selectedMemberGroupIndex = -1;
    selectedMemberIndex = -1;

    this.topResult = topResult;
    this.selection = topResult;
}
```